src/components/EditOrganization/index.jsx

```diff
@@ -0,0 +1,222 @@
+import React, { useEffect, useRef, useState } from "react";
+import "./style.css";
+import {
+  Form,
+  Input,
+  Upload,
+  Button,
+  Typography,
+  Divider,
+  Spin,
+  message,
+} from "antd";
+import { InboxOutlined } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import firebase from "../../firebaseConfig";
+import "firebase/storage";
+
+const db = firebase.firestore();
+
+const storage = firebase.storage();
+
+const storageRef = storage.ref(); // change the storage rule to "request.auth != null" once auth is setup
+
+const { Title } = Typography;
+const { Dragger } = Upload;
+
+const EditOrganization = ({ isTesting }) => {
+  const { t } = useTranslation();
+
+  const [organization, setOrganization] = useState({});
+
+  const [loading, setLoading] = useState(true);
+
+  const folderName = useRef(null);
+
```

Comment:
new lines between consts again here

Reply:
What's the general rule for new lines? I kind of make mistakes with it every time.